packages/mc-scripts/src/commands/login.js

```diff
@@ -0,0 +1,98 @@
+const path = require('path');
+const { promisify } = require('util');
+const read = promisify(require('read'));
+const fetch = require('node-fetch');
+const fs = require('fs');
+const homedir = require('os').homedir();
+const { processConfig } = require('@commercetools-frontend/application-config');
+const { MC_API_URLS, DEFAULT_CREDENTIALS } = require('../constants');
+
+function validateEmail(email) {
+  const re = /\S+@\S+\.\S+/;
+  return re.test(email);
+}
+
+const getEmail = async () => {
+  let email = await read({ prompt: 'Email: ' });
+  if (!validateEmail(email)) {
+    console.log('Please input a valid email');
+    email = await getEmail();
+  }
+  return email;
+};
+
+const getPassword = async () => {
+  return await read({ prompt: 'Password: ', silent: true });
+};
+
+const getUpdatedCredentials = (oldCredentials, cloudIdentifier, token) => {
+  oldCredentials[cloudIdentifier] = token;
+  return JSON.stringify(oldCredentials);
+};
+
+const updateCredential = (cloudIdentifier, token) => {
+  const dirpath = `${homedir}/.mcscriptsrc`;
+  const filename = 'credentials.json';
+  const filePath = `${dirpath}/${filename}`;
+
+  try {
+    if (!fs.existsSync(dirpath)) {
+      fs.mkdirSync(dirpath);
+      const credentials = getUpdatedCredentials(
+        DEFAULT_CREDENTIALS,
+        cloudIdentifier,
+        token
+      );
+      console.log({ credentials });
+      fs.writeFileSync(filePath, credentials);
+    } else {
+      const data = fs.readFileSync(filePath, 'utf8');
+      console.log({ data });
+      const credentials = getUpdatedCredentials(
+        JSON.parse(data),
+        cloudIdentifier,
+        token
+      );
+      fs.writeFileSync(filePath, credentials);
+    }
+  } catch (err) {
+    console.error('An error occurred');
+  }
+};
+
+const login = async () => {
+  const applicationConfig = processConfig({
+    applicationPath: path.join(__dirname, 'fixtures'),
+  });
+  const cloudIdentifier = applicationConfig.env.cloudIdentifier.toUpperCase();
+  const email = await getEmail();
+  const password = await getPassword();
```

Comment:
In case there is already a token saved, we probably don't need to log in again.

We can also check if the token is still valid and only in case it's not, we should log the user in again.

Can also be a follow up, just wanted to share this first.